### PR TITLE
Remove kubernetes

### PR DESCRIPTION
**Pull request by moleske**
Thanks for contributing to cloud_controller_ng. To speed up the process of reviewing your pull request please provide us with:

* A short explanation of the proposed change:
Since [capi-k8s-release](https://github.com/cloudfoundry/capi-k8s-release) and [cf-for-k8s](https://github.com/cloudfoundry/cf-for-k8s) are both deprecated and not supported because of [korfi](https://github.com/cloudfoundry/korifi/), it seemed like a reasonable thing to remove kubernetes related code to reduce complexity (maybe)

I didn't touch any models cause that involved more thought than I wanted to do for this experiment.  This all is like big breaking changes anyway.

This was not done with the idea "we should quickly go remove k8s things" but more of a "I wonder if we could"

* An explanation of the use cases your change solves
So I don't have to think about kpack or other things when working cloud controller in a bosh/vms world

* Links to any other associated PRs
[capi-release pr](https://github.com/cloudfoundry/capi-release/pull/287)

* [x] I have reviewed the [contributing guide](https://github.com/cloudfoundry/cloud_controller_ng/blob/main/CONTRIBUTING.md)

* [x] I have viewed, signed, and submitted the Contributor License Agreement

* [x] I have made this pull request to the `main` branch

* [x] I have run all the unit tests using `bundle exec rake`

* [x] I have run [CF Acceptance Tests](https://github.com/cloudfoundry/cloud_controller_ng/blob/main/spec/README.md#cf-acceptance-tests-cats)

passing run of CATS version 12.1.0 on a bosh lite with the following config
```json
{
  "admin_user": "admin",
  "skip_ssl_validation": true,
  "use_http": true,
  "include_apps": true,
  "include_capi_experimental": true,
  "include_detect": true,
  "include_security_groups": true,
  "include_services": true,
  "include_v3": true,
  "include_tasks": true,
  "include_backend_compatibility": false,
  "include_capi_no_bridge": false,
  "include_container_networking": false,
  "include_credhub" : false,
  "include_docker": false,
  "include_internet_dependent": false,
  "include_isolation_segments": false,
  "include_persistent_app": false,
  "include_private_docker_registry": false,
  "include_privileged_container_support": false,
  "include_route_services": false,
  "include_routing": false,
  "include_routing_isolation_segments": false,
  "include_service_discovery": false,
  "include_service_instance_sharing": false,
  "include_ssh": false,
  "include_sso": false,
  "include_zipkin": false
}
```

<details>
<summary>older run of CATS version 9.5.0 with the following config</summary>
```json
{
  "skip_ssl_validation": true,
  "use_http": true,
  "include_apps": true,
  "include_capi_experimental": true,
  "include_detect": true,
  "include_security_groups": true,
  "include_services": true,
  "include_v3": true,
  "include_tasks": true,
  "include_backend_compatibility": false,
  "include_capi_no_bridge": false,
  "include_container_networking": false,
  "include_credhub" : false,
  "include_docker": false,
  "include_internet_dependent": false,
  "include_isolation_segments": false,
  "include_persistent_app": false,
  "include_private_docker_registry": false,
  "include_privileged_container_support": false,
  "include_route_services": false,
  "include_routing": false,
  "include_routing_isolation_segments": false,
  "include_service_discovery": false,
  "include_service_instance_sharing": false,
  "include_ssh": false,
  "include_sso": false,
  "include_zipkin": false
}
```
and had the following failure on a bosh-lite but I didn't retry cause my day ended
```bash
Summarizing 1 Failure:
  [FAIL] [services] Service Instance Lifecycle Synchronous operations when there is an app [BeforeEach] bindings can bind service to app and send arbitrary params
  /home/pivotal/workspace/cf-acceptance-tests/services/service_instance_lifecycle.go:545

Ran 125 of 233 Specs in 3171.070 seconds
FAIL! -- 124 Passed | 1 Failed | 2 Pending | 106 Skipped
```
</details>

### CODE REVIEW

**Comment by moleske** [comment]
I'm marking this ready as cats is passing on a bosh lite.  I'm not particularly positive we should merge as is or if there's a smaller way to do this.  I'd love thoughts on breaking change concerns for thise

File: app/models/runtime/droplet_model.rb [hunk 1]
```diff
@@ -89,12 +89,8 @@ def docker?
       lifecycle_type == DockerLifecycleDataModel::LIFECYCLE_TYPE
     end
 
-    def kpack?
-      lifecycle_type == KpackLifecycleDataModel::LIFECYCLE_TYPE
-    end
-
     def has_docker_image?
-      docker? || kpack?
+      docker?
```
Comment (Gerg) [inline app/models/runtime/droplet_model.rb:93]: This method can probably go away or become an alias.

**Comment by tcdowney** [comment]
@braunsonm y'all aren't using this right?

**Comment by braunsonm** [comment]
Yea @tcdowney but our version is fairly out of date at this point. We've known this wasn't being maintained for awhile now so understandable that it's getting removed.

So long as you aren't planning on removing the old release images or anything we should be fine.

Appreciate the call out.

**Comment by sethboyles** [comment]
@tcdowney this also removes OPI/Eirini code--do you know the status of that project?

**Comment by tcdowney** [comment]
> @tcdowney this also removes OPI/Eirini code--do you know the status of that project?

It's no longer under active development, but may still be under some light maintenance? Development work was shifted over time from `eirini` -> `eirini-controller` -> `korifi`. @gcapizzi is one of the WG leads for it so he might be able to answer better.

**Comment by gcapizzi** [comment]
Eirini is only used in `cf-for-k8s`, which I think has been deprecated and archived, so I think it's safe to get rid of the Eirini stuff from CC.

**Review by sethboyles — APPROVED** [review]

**Comment by mkocher** [comment]
Nice, I've come across this code when making changes to cloud controller. Thanks for cleaning it up!

**Comment by sethboyles** [comment]
@moleske I say let's merge this!

**Review by tcdowney — APPROVED** [review]
Goodbye Kubernetes 🥲

**Pull request merged**